mock-core-configs/etc/mock/epel-8-s390x.cfg

```diff
@@ -0,0 +1,5 @@
+include('templates/epel-8.tpl')
```

Comment:
You are missing: `include('templates/centos-8.tpl')` at very top of this file.

Reply:
Hmm, even with that line it does not work. Without the centos line I get:
```

TypeError: can only concatenate tuple (not "str") to tuple

ERROR: Error in configuration
```
With the line I get:
```
CentOS-8 - Base                                                                          400  B/s |  38  B     00:00    
Error: Failed to download metadata for repo 'BaseOS': Cannot prepare internal mirrorlist: No URLs in mirrorlist
ERROR: Command failed: 
 # /usr/bin/dnf --installroot /var/lib/mock/epel-8-s390x-bootstrap/root/ --releasever 8 --setopt=deltarpm=False --allowerasing --disableplugin=local --disableplugin=spacewalk --forcearch s390x install dnf dnf-plugins-core --setopt=tsflags=nocontexts
No matches found for the following disable plugin patterns: local, spacewalk
CentOS-8 - Base                                                                          400  B/s |  38  B     00:00    
Error: Failed to download metadata for repo 'BaseOS': Cannot prepare internal mirrorlist: No URLs in mirrorlist
```